### ISSUE TITLE
Convert README to markdown

### ISSUE THREAD

**Issue by YurySolovyov** [documentcloud/closure-compiler#49]
@knowtheory is there any specific reason current readme is textile and not md?
Any thoughts on converting it to markdown (I can do the job)?

**Comment by knowtheory** [comment]
Yeah, there isn't a reason we've held back and i personally much prefer markdown, so go for it!
